src/agent/state.ts

```diff
@@ -154,7 +154,7 @@ class StateResolver {
     // Evaluate the watch expressions
     const evalIndexSet = new Set();
     if (that.expressions_) {
-      that.expressions_.forEach(function(expression, index) {
+      that.expressions_.forEach(function(expression, index2) {
```

Comment:
It could be helpful to have the first occurrence of `index` changed to `index1` too.